qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -430,6 +460,26 @@ def init():
     webenginequtescheme.init()
     spell.init()
 
+    # For some reason we need to keep a reference, otherwise the scheme handler
+    # won't work...
+    # https://www.riverbankcomputing.com/pipermail/pyqt/2016-September/038075.html
```

Comment:
Not really accurate anymore, we need to keep a reference for other reasons as well now.

Reply:
I wouldn't remove this in case we find a better way to structure all of the settings/profiles stuff in the future and the *new* reason to keep a reference dissapears.